### DISCUSSION TITLE
ImportError for lemmy.pipe

### DISCUSSION THREAD

**Discussion by astasofie** [explosion/spaCy#11009] [Help: Coding & Implementations] [answered]
Hi smart people,

I'm trying to use Lemmy as a SpaCy integration following the instructions [here.](https://spacy.io/universe/project/lemmy) However, when I try running import lemmy.pipe I get the error **ImportError:** cannot import name 'PRON_LEMMA' from 'spacy.symbols' . 

I'm at a loss - how do I get around this? I can't even find out what the exact issue is so finding a solution is tricky.

I'm using python 3.8 and SpaCy 3.3.0 with the [DaCy pipeline](https://pythonawesome.com/dacy-the-state-of-the-art-danish-nlp-pipeline-using-spacy/). 

Thank you for you input!

**Comment by adrianeboyd** [comment, marked as answer]
This package hasn't been updated in a while and it looks like it only supports spacy v2.

It probably wouldn't be difficult to update it for spacy v3 if you'd like to use with the DaCy pipelines. It would require the changes mentioned for "stateful class components" described in this upgrading guide: https://spacy.io/usage/v3#migrating-pipeline-components

**Comment by vrunm** [comment]
@adrianeboyd After downgrading to spacy 2.1.0 it still gives the same import error. Is there any way around this import?

**Reply by rmitsch** [reply]
Please don't (re-)open duplicate discussions. Pinging maintainers is also not necessary, we get notified about new replies. Let's continue this conversation in https://github.com/explosion/spaCy/discussions/12224.

**Reply by vrunm** [reply]
Sure will do that.
